### DISCUSSION TITLE
Query Log have unusual Requests.    xxx.lan

### DISCUSSION THREAD

**Discussion by Gangnamsstyle** [AdguardTeam/AdGuardHome#2937] [Q&A]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [ ] I am running the latest version Yes.
- [ ] I checked the documentation and found no answer Yes.
- [ ] I checked to make sure that this issue has not already been filed Yes.

### Issue Details

* **Version of AdGuard Home server:**
  * 
* **How did you install AdGuard Home:**
  * 
* **How did you setup DNS configuration:**
  * 
* **If it's a router or IoT, please write device model:**
  * 
* **CPU architecture:**
  * 
* **Operating system and version:**
  * 

### Screenshots

<details><summary>Screenshot:</summary>

</details>

### Additional Information

Hello,When I use google chrome ,I serch something .I show in ther picture below.Sometimes google search not work, no connection.But I can ping to the google sever.It means that internet no problems.Is it just dns problesm that Chrome can not find correct dns ?

![google](https://user-images.githubusercontent.com/69400033/114114115-b6c8b500-9912-11eb-80c3-53fdc5f36df2.png)
 
I also found Query Log in the Adgurad Home have have unusual Requests.  
 xxx.lan
![image](https://user-images.githubusercontent.com/69400033/114114265-1d4dd300-9913-11eb-8f96-d26c921dea35.png)

![image](https://user-images.githubusercontent.com/69400033/114114284-25a60e00-9913-11eb-8170-a0c9017e2e4d.png)

The requests xxx.lan is the name I search in google chrome.

any help ,thank you a lot.

**Comment by ainar-g** [comment]
Hi.  It's probably because Google Chrome performs a DNS query for anything you enter into the address bar.  Browsers usually make up to four requests: `A` and `AAAA` for `thing` and the same two for `thing.local`.  If you set your local TLD to `.lan` instead of `.local` somewhere in your system then Google Chrome probably decided to use that domain instead of `.local`.

This behaviour can probably be disabled somewhere in Google Chrome's settings.
